### PR TITLE
Breadcrumbs: fix styles when hover on links. It will be better when you move breadcrumbs after main menu line

### PR DESCRIPTION
**Pull request by rtershak**
https://trello.com/c/YRgp0aPf/53-breadcrumbs-fix-styles-when-hover-on-links-it-will-be-better-when-you-move-breadcrumbs-after-main-menu-line

Screen: http://www.awesomescreenshot.com/image/866105/604f22eea38dbad83291b69a4487c7c8

code review:
- [ ] @CHILL99 
- [ ] @DimaSmagulov 
- [ ] @krisberry 
- [ ] @natalya5 
- [x] @oleg08 
- [ ] @OrestF 
- [x] @vvs79 

2nd lvl review:

- [x] @Kmalynss
- [ ] @serhiym 
- [ ] @ruba-ruba

### CODE REVIEW

**Comment by oleg08** [comment]
reviewed

File: app/views/layouts/_header.html.slim [hunk 1]
```diff
@@ -24,3 +22,7 @@ nav.navbar.navbar-default.navbar-fixed-top.topnav[role="navigation"]
         - else
           li
             = link_to('Login', new_user_session_path)
+- if user_signed_in?
+  .navbar-brand.topnav.breadcrumb
+    = render_breadcrumbs separator: " / "
+
```
Comment (Kmalynss) [inline app/views/layouts/_header.html.slim]: remove spaces

**Comment by OrestF** [comment]
Wow!
Breadcrumbs style is't look good for me. It fall out from application style.

File: app/views/layouts/_header.html.slim (hunk 1 above)
Comment (DimaSmagulov) [inline app/views/layouts/_header.html.slim]: I think the main idea was to get breadcrums out from the header, and now we just have 2 navbars in header, it's not good.

File: app/assets/stylesheets/custom.css.scss [hunk 2]
```diff
@@ -244,6 +243,8 @@ td {
     color:white;
     text-decoration: none;
     }
+  margin: 5px 0 0 100px;
+  background: #000000;
```
Comment (DimaSmagulov) [inline app/assets/stylesheets/custom.css.scss]: I think we dont need beckground for breadcrumbs (especially black)

File: app/views/layouts/_header.html.slim (hunk 1 above)
Comment (DimaSmagulov) [inline app/views/layouts/_header.html.slim]: I think the main idea was to get breadcrumbs out of _header and now we just have 2 navbars in header, it's not good.

**Comment by vvs79** [comment]
reviewed

**Comment by Kmalynss** [comment]
gtg

**Pull request merged**